AuthenticationExample/AuthenticationExample/AuthenticationApp.swift

```diff
@@ -48,9 +38,9 @@ struct AuthenticationApp: App {
             // - Integrated Windows Authentication (IWA)
             // - Client Certificate (PKI)
             .authenticator(authenticator)
-            .environmentObject(authenticator)
```

Comment:
While considering [this feedback](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/392#discussion_r1295011502) I noticed that this environment object goes totally unused in the example.

Without a good use for the suggested environment value it seems like it might be better to just remove it all together.